src/basic/cgroup-util.c

```diff
@@ -2358,6 +2358,9 @@ int cg_mask_supported(CGroupMask *ret) {
                  * everything else off. */
                 mask &= CGROUP_MASK_CPU | CGROUP_MASK_MEMORY | CGROUP_MASK_IO | CGROUP_MASK_PIDS;
 
+                /* Device controller is BPF-based on cgroup v2 */
+                mask |= CGROUP_MASK_DEVICES;
```

Comment:
ah, weshouldn't do this either, i missed that in the earlier review. Let's not fake the "devices" controller, we don't have it. I think we shouldn't pretend we had a controller we don't have...

Reply:
Should then "if ((apply_mask & CGROUP_MASK_DEVICES))" be converted to something else? Should I add "&& !cg_all_unified()"?
Btw, all kernel docs are describing this functionality as "device controller".